.github/workflows/examples.yaml

```diff
@@ -0,0 +1,109 @@
+name: examples
+
+on:
+  push:
+    paths:
+    - 'examples/**'
+    - 'packages/**'
+  pull_request:
+    paths:
+    - 'examples/**'
+    - 'packages/**'
+
+env:
+  flutter_channel: 'stable'
+  flutter_version: '1.12.13+hotfix.7'
```

Comment:
can we have a specified `flutter_version` and `flutter_channel` so that we don't need to update it in multiple places (main workflow and examples workflow)?

Reply:
Ehh, probably not. I mean we could probably fetch that using bash task inside of each job, but...

Reply:
Great suggestion!
`subosito/flutter-action@v1` action uses variables in the github action workflow definition, so we may have to deviate a bit from its interface if adding a bash task.
I don't have strong opinion on this, @felangel do you like adding a bash task option?